charts/minibroker/templates/deployment.yaml

```diff
@@ -49,12 +49,10 @@ spec:
         ports:
         - containerPort: 8080
       - name: tiller
-        image: "gcr.io/kubernetes-helm/tiller:v2.13.1"
+        image: "{{ .Values.kube.registry.hostname }}/{{ .Values.kube.organization }}/helm-tiller:2.14.2"
         imagePullPolicy: IfNotPresent
         env:
         - name: TILLER_NAMESPACE
           value: {{ .Release.Namespace }}
         - name: TILLER_HISTORY_MAX
           value: "1"
-        command: ["/tiller"]
-        args: ["--listen=localhost:44134"]
```

Comment:
@bisingh Did you verify that the `gcr.io/kubernetes-helm/tiller:v2.13.1` image still works, even without the `--listen=localhost:44134` argument?

Reply:
I have verified that the minibroker and tiller container were up and running with these changes:

```
vagrant@vagrant-kube:~/minibroker> k logs minibroker-minibroker-69ff7db7d5-r28t5 -n minibroker -c minibroker
kubectl logs minibroker-minibroker-69ff7db7d5-r28t5 -n minibroker -c minibroker
I1003 17:33:08.160659       1 minibroker.go:81] namespace: minibroker
I1003 17:33:08.160710       1 helm.go:39] Helm Home: /root/.helm
I1003 17:33:09.337245       1 main.go:92] Starting broker!
I1003 17:33:09.337262       1 server.go:110] Starting server on :8080
```
```
vagrant@vagrant-kube:~/minibroker> k logs minibroker-minibroker-69ff7db7d5-r28t5 -n minibroker -c tiller
kubectl logs minibroker-minibroker-69ff7db7d5-r28t5 -n minibroker -c tiller
[main] 2019/10/03 17:33:08 Starting Tiller v2.14.2 (tls=false)
[main] 2019/10/03 17:33:08 GRPC listening on localhost:44134
[main] 2019/10/03 17:33:08 Probes listening on :44135
[main] 2019/10/03 17:33:08 Storage driver is ConfigMap
[main] 2019/10/03 17:33:08 Max history per release is 1
```

Reply:
Ok, thanks! And sorry for my confused question; I somehow assumed that we would still support using the old image as well, but we don't have any way to configure the imagename and tag, so it doesn't really matter.